content/en/integrations/guide/source-code-integration.md

```diff
@@ -175,23 +116,92 @@ If your build process is executed in CI within a Docker container, perform the f
+| Datadog Serverless Framework Plugin | If you are using the [Datadog Serverless Plugin][1] to instrument your serverless application, use a serverless-plugin-datadog `version >= 5.18.0`.                                                                         |
+| datadog-cdk-constructs              | If you are using the [Datadog CDK Construct][2] to instrument your serverless application, use a datadog-cdk-constructs `version >= 0.8.5` for AWS CDK v1, and datadog-cdk-constructs-v2 `version >= 1.4.0` for AWS CDK v2. |
+| datadog-ci                          | If you are using the [Datadog CLI client][3] to instrument your serverless application, use a datadog-ci `version >= 2.4.1`. You must run the CLI tool in the same directory as the code repository.                        |
+
+[1]: /serverless/libraries_integrations/plugin/
+[2]: /serverless/libraries_integrations/cdk/
+[3]: https://www.npmjs.com/package/@datadog/datadog-ci
+[4]: /serverless/aws_lambda/configuration/
+
+{{% /tab %}}
+{{% tab "Host" %}}
+
+To tag your traces, spans, and profiles with `git.commit.sha` and `git.repository_url`, configure the tracer with the `DD_TAGS` environment variable:
+
+```
+export DD_TAGS="git.commit.sha:<FULL_GIT_COMMIT_SHA>,git.repository_url:git-provider.example/me/my-repo"
+./my-application start
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+## Synchronize your repository metadata
+
+To link your telemetry with source code, your repository metadata needs to be synchronized to Datadog.
+
+<div class="alert alert-info">
+Datadog doesn't store the actual content of files in your repository, only Git commit and tree objects.
+</div>
 
 {{< tabs >}}
 {{% tab "GitHub" %}}
 
-Install Datadog's [GitHub integration][1] on the [GitHub integration tile][2] to link your telemetry with your source code. When specifying permissions on the integration tile, enable Datadog read permissions to **Contents**.
+Install Datadog's [GitHub integration][1] on the [GitHub integration tile][2] to let Datadog synchronize your repository metadata automatically. When specifying permissions on the integration tile, select at least **Read** permissions for **Contents**.
 
-By setting up the GitHub integration, you can see inline code snippets in **Error Tracking**. For more information, see [Inline Source Code](#inline-source-code).
+Setting up the GitHub integration also allows you to see inline code snippets in **Error Tracking**.
 
 [1]: https://docs.datadoghq.com/integrations/github/
 [2]: https://app.datadoghq.com/integrations/github/
 {{% /tab %}}
 {{% tab "Other Git Providers" %}}
 
 <div class="alert alert-warning">
-Self-hosted instances or private URLs are not supported.
+Repositories on self-hosted instances or private URLs are not supported out-of-the-box by the Source Code Integration. Please reach out to support.
```

Comment:
Can you clarify what the line about contacting Support means? Are we instructing users to contact Support when they want to use repositories on self-hosted instances or private URLs (because they're not supported out-of-the-box and thus require additional configuration/steps on our end)?

Reply:
Exactly, we currently don't have a way to configure that in-app, but we can configure it manually for them if they reach out to us

Reply:
Thanks for the headsup ! I updated the page.

Reply:
Perfect, thanks! This looks good from my end. Is this ready to be merged now?

Reply:
I'd like to wait for a teammate of mine to double check everything makes sense, if it's possible to hold off for a bit